client/src/components/mastheadOptions/mastheadOptions.js

```diff
@@ -3,33 +3,15 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import { Dropdown, Icon, MenuItem } from 'patternfly-react';
-
-import Store from '../../redux/store';
-import { getUser, logoutUser } from '../../redux/actions/userActions';
-import { aboutTypes } from '../../redux/constants';
-import helpers from '../../common/helpers';
+import { getUser } from '../../redux/actions/userActions';
 
 class MastheadOptions extends React.Component {
```

Comment:
Note: At some point we could probably go full stateless with this component, turn it into a ```const``` instead of an ecma ```class```.